app/src/main/res/layout/fragment_login_screen.xml

```diff
@@ -0,0 +1,207 @@
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:fontFamily="@font/poppins"
+        android:gravity="center"
+        android:text="@string/sign_in"
+        android:textColor="@color/Secondary"
+        android:textSize="@dimen/fontHeading1"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginTop="32dp"
+        android:layout_marginBottom="-32dp"
+        app:cardCornerRadius="32dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView2" >
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginBottom="32dp">
+
+            <Button
+                android:id="@+id/btn_google"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="24dp"
+                android:layout_marginTop="24dp"
+                android:layout_marginEnd="12dp"
+                android:backgroundTint="@color/btnColor"
+                android:textSize="@dimen/fontHeading1"
+                app:cornerRadius="8dp"
+                app:icon="@drawable/google_icon"
+                app:iconGravity="textEnd"
+                app:iconTint="@color/Secondary"
+                app:layout_constraintEnd_toStartOf="@id/btn_facebook"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:ignore="SpeakableTextPresentCheck" />
+
+            <Button
+                android:id="@+id/btn_facebook"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="12dp"
+                android:layout_marginTop="24dp"
+                android:layout_marginEnd="24dp"
+                android:backgroundTint="@color/btnColor"
+                android:textSize="@dimen/fontHeading1"
+                app:cornerRadius="8dp"
+                app:icon="@drawable/facebook_icon"
+                app:iconGravity="textEnd"
+                app:iconTint="@color/Secondary"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@id/btn_google"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:ignore="SpeakableTextPresentCheck" />
+
+            <TextView
```

Comment:
fontFamily, textSize, textStyle and textColor attributes has been called more the once with the same values for this textview and textview at line #117
To reduce the code duplication and make the views much easier to modify you should create custom theme for the view, you can read more here: https://developer.android.com/guide/topics/ui/look-and-feel/themes

Reply:
Done